.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -378,22 +390,34 @@ and Linux `tsh` binaries are tested for FIDO2 items.
 
 - [ ] Diagnostics
 
-    Both commands should pass all tests.
+    Commands should pass all tests.
 
   - [ ] `tsh fido2 diag`
   - [ ] `tsh touchid diag`
+  - [ ] `tsh webauthnwin diag`
 
 - [ ] Registration
   - [ ] Register a passworldess FIDO2 key (`tsh mfa add`, choose WEBAUTHN and
         passwordless)
   - [ ] Register a Touch ID credential (`tsh mfa add`, choose TOUCHID)
+  - [ ] Register a passworldess FIDO2 key (`tsh mfa add`, choose WEBAUTHN and
```

Comment:
Ditto - say "Touch ID / Windows Hello" in the existing items?

At the top of this section we say:

"Ideally both macOS and Linux `tsh` binaries are tested for FIDO2 items."

We could change that to:

"`tsh` binaries for each operating system (Linux, macOS and Windows) must be tested separately for FIDO2 items."